tools/complete_input.json

```diff
@@ -1,7 +1,7 @@
 {
     "output": {
         "directory": "./",
-        "output-format": "HDF5",
```

Comment:
This needs to be added back, complete_input.json is pretty useful.

Reply:
thanks for spotting it. Re-added